composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
+        "league/climate": "^2.0 || ^3.0",
         "symfony/console": "^2.0",
         "symfony/process": "^2.1"
     },
 
     "require-dev": {
         "mockery/mockery": "^0.9",
-        "phpunit/phpunit": "^4.0",
+        "phpunit/phpunit": "^5.1",
```

Comment:
I would like to see this be `^4.0|^5.0` for maximum backwards compatibility when installed as a dependency of a project.